spec/system/user_preferences_navigation_spec.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+describe 'Redesigned user page navigation menu', type: :system, js: true do
+  fab!(:user) { Fabricate(:user) }
+  let(:everyone_group) { Group[:everyone] }
+
+  describe "when visiting the user's preferences page with redesigned user page nav enabled" do
```

Comment:
Just a token test for now because I don't think the more complex scrolling behaviours are worth testing at this point in time. If the implementation turns out to be brittle, we at least have the system test setup for us to write tests.

Reply:
Yes makes sense :+1: